collector/loadavg.go

```diff
@@ -3,18 +3,15 @@
 package collector
 
 import (
+	"errors"
 	"fmt"
-	"io/ioutil"
-	"strconv"
-	"strings"
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/log"
 )
 
-const (
-	procLoad = "/proc/loadavg"
-)
+// #include <stdlib.h>
```

Comment:
(The reason was that cgo causes problems with cross-compilation (which might get better with Go 1.5), and also draws in more shared libraries)